src/spacy/access.clj

```diff
@@ -0,0 +1,96 @@
+(ns spacy.access
+  (:require [buddy.sign.jwt :as jwt]
+            [schema.core :as schema]
+            [clojure.tools.logging :as log]
+            [net.cgrand.enlive-html :as html]
+            [yada.yada :as yada]
+            [yada.security]
+            [spacy.config :as config]
+            [environ.core :as env]))
+
+(def jwt-key (or (env/env :jwt-key)
+                 "UAWX9WK366IRDF5AQTRPX2I457AFTTU93PBB1QAJME107IKJM4A5HPW2I6PJRHB0"))
```

Comment:
@jstepien What do you think about this method of configuring the JWT key?

I'm not _too_ worried about the integrity of our JWT key at this point, since we recommend doing the real authentication in a sidecard anyway.

Reply:
It would probably be possible to add a

```
{:access {:jwt-key #or [#env JWT_KEY "UAWX9WK366IRDF5AQTRPX2I457AFTTU93PBB1QAJME107IKJM4A5HPW2I6PJRHB0"]}}
```

entry to our `config.edn` which I personally think would be preferable (all the config in one place).

But when defining this `jwt-key` variable, I don't know which `:profile` is currently running, so that makes it a bit more inconvenient.

Reply:
I agree; `config.edn` seems to be the right place for it. Still, I guess the environ-based solution will suffice for now. We can replace it with a config-based approach later, when we know more about our authentication needs.

Reply:
I changed it in 777b1aeb46ee0d9ffa8bb3e1b7bfa5350aa86f8d to use `config.edn`, but the `jwt-key` just always accesses the config object with profile `:prod` so that we don't have to access our system map to retrieve the config.

Reply:
Sounds good! That should do it for now.